hub/hub.html

```diff
@@ -52,6 +52,10 @@ <h3 class="research-hub-sub-title">Explore and extend models<br> from the latest
               </div>
             </div>
           -->
+            <div id="hub-icons" class="hub-select-container col-5">
+              <a  href="{{ site.baseurl }}"><img src="/assets/images/full-hub-icon-selected.svg"></a>
+              <a class="compact-hub-icon" href="{{ site.baseurl }}research-models/compact"><img src="/assets/images/compact-hub-icon.svg"></a>
```

Comment:
I tried to deploy this to netlify, and the compact link didn't work. It should look like: `{{ site.baseurl }}/hub/research-models/compact`.